dashboard/test/models/script_test.rb

```diff
@@ -253,4 +253,28 @@ def setup
     assert !Script.find_by_name('flappy').pd?
     assert Script.find_by_name('ECSPD').pd?
   end
+
+  test 'hoc?' do
+    assert Script.find_by_name('flappy').hoc?
+    assert Script.find_by_name('mc').hoc?
+    assert Script.find_by_name('hourofcode').hoc?
+    assert Script.find_by_name('Hour of Code').hoc?
+    assert Script.find_by_name('frozen').hoc?
+    assert Script.find_by_name('playlab').hoc?
+    assert_not Script.find_by_name('20-hour').hoc?
+    assert_not Script.find_by_name('course1').hoc?
+    assert_not Script.find_by_name('course2').hoc?
+    assert_not Script.find_by_name('course3').hoc?
+    assert_not Script.find_by_name('course4').hoc?
+  end
+
+  test 'minecraft?' do
+    assert_not Script.find_by_name('flappy').minecraft?
+    assert Script.find_by_name('mc').minecraft?
+  end
+
+  test 'twenty_hour?' do
```

Comment:
Can you add a test that verifies that the literal name of the high scale HOC scripts has not changed so that we know our config keys haven't changed?  The names should be 'starwars', 'starwarsblocks', 'mc', 'hourofcode', and 'frozen', and 'flappy'.